### DISCUSSION TITLE
How to start: utilizzo Home Assistant per integrazione home kit e smartthing

### DISCUSSION THREAD

**Discussion by giuseppe-mancuso** [indomus/forum#3186] [HUB: Home Assistant]
Ciao a tutti e prima di tutto complimenti per i mille spunti e la professionalità dimostrata nel sito e nel forum.
Veniamo al dilemma: sto cercando di sistemare il mio primo hub domestico; per scelta in casa poi sempre usato dispositivi compatibili con HomeKit Apple (sono un Mac user di vecchia data e ho sempre avuto dispositivi tra loro compatibili) fino a dove possibile. Alcuni elettrodomestici (tv, lavatrice, condizionatori) li ho comprati samsung per la qualità il costo ecc.
Sorge quindi il problema di far parlare i due mondi e credo che Home Assistant sia la soluzione migliore ma qui sorge il problema: fermo restando che la soluzione MiniPC è tecnicamente la migliore ma per avere come unico scopo il far parlare questi due mondi è Comunque consigliabile andare in quella direzione? in termini di costi/benefici una soluzione meno potente ma più economica potrebbe aver senso (home assististant Green?)

Grazie mille a tutti

Giuseppe

**Comment by Scialla** [comment]
In passato ho fatto lo stesso ragionamento affidandomi a HomeBridge per rendere compatibili Homekit i dispositivi che non lo erano poi..... sono passato a HomeAssistant e mi sono reso conto di quanto limitato fosse Homekit per gestire la propria domotica. Il passaggio l'ho fatto quando volevo gestire i carichi di casa per non far saltare il contatore, cosa impossibile, all'epoca, da fare con homekit, e mi si è aperto un mondo.... tutte le automazioni fatte su homekit erano (e sono) una merda in confronto a quelle fatte da HA.
Tutta questa premessa solo per farti capire che, una volta che inizierai ad usare HA, abbandonerai Homekit e, quindi, per rispondere alla tua domanda, dopo ti servirà il miniPC
Sconsiglio il resto, a meno che non vuoi iniziare a giocare ma, per quello, puoi usare qualcosa che magari hai già a casa, oppure prenderti un Raspberry usato. Tieni presente che se vai su HA core, puoi iniziare a giocare anche con un RPi3b ma, se vai su HAOS, meglio stare su un 4.
Io uso Core e, tempo fa, sono passato da un RPi3b a un MiniPC con processore Atom, una porcheria di parecchi anni fà che è tuttora più che sufficiente per gestire tutto..
Certo che se, in futuro, vorrai integrare telecamere, servirà qualcosa di più performante (io ne gestisco 4 ma senza alcuna decodifica ne analisi flusso)
PS lascia perdere Homekit che è veramente limitato, potrai sempre esporre i tuoi device da HA a Homekit e vederli anche lì (io lo faccio ma praticamente Homekit non lo apro neanche più)

**Reply by giuseppe-mancuso** [reply]
Grazie mille per la risposta e il consiglio: ciò che mi "preoccupa" è l'avere un sistema (anche con videocamere) già tutto operativo dentro HomeKit e soprattutto la facilità d'uso per altri familiari (mia moglie usa HomeKit spesso per tante piccole cose e non vorrei che con HA sia tutto più complesso). Da questo il desiderio di mantenere HK.
Detto ciò da quanto dici la soluzione potrebbe essere Comunque gestire tutto ex-novo con HA e poi usare HK solo di supporto/appoggio.

**Reply by Scialla** [reply]
Esatto! La libertà di creare automazioni che ti da HA è IMPOSSIBILE da ottenere con Homekit. Puoi usarlo però, almeno inizialmente o per i tuoi familiari, come interfaccia grafica per gestire la tua domotica demandando ad HA tutto il resto ed avendo così la libertà di fare tutto quello che vuoi interfacciandoti anche con altre marche e, soprattutto, altri standard

**Reply by Marcunz** [reply]
Per Scialla: situazione molto simile a Giuseppe, soprattutto sulla necessità di rendere facile il tutto agli altri familiari mac-user, per rimanere in ambito mac è pensabile usare un vecchio Mac Mini? O per forza (è meglio) un mini pc?

Tu Giuseppe cosa hai usato come telecamere *esterne* per rimanere in HK (e magari usare Secure video di HK) ? Hai interfacciato un NVR o direttamente le telecamere?

**Reply by Scialla** [reply]
Certo che puoi usare un vecchio MacMini ma, per i consumi, non è consigliabile... tieni presente che è una macchina che deve stare accesa H24 7/7gg, il MacMini consuma troppo e comunque è sprecato da dedicare tutto ad HA a meno che non vuoi gestire telecamere ma, nel caso, molto meglio l'uso di NVR da integrare poi in HA ed eventualmente esporre verso Homekit (anche con Secure video ma su questo, nel caso, può spiegarti meglio @NdR91)

**Reply by giuseppe-mancuso** [reply]
Ho banalmente usato le camere netatmo che nascono con supporto natio ad HK.

@Scialla domanda forse banale: ma nella configurazione iniziale faresti passare tutto da HA per avere massimo controllo giusto?
In questo caso, se mancasse la corrente e il pc non avesse un gruppo di continuità, perderei tutti i controlli a distanza giusto?

**Reply by Scialla** [reply]
Beh certo, ma è così anche adesso, a meno che non hai la AppleTV (che fa da HUB) sotto UPS insieme a tutte le cose che vuoi controllare da remoto

**Reply by giuseppe-mancuso** [reply]
Chiaro, nel momento che salta la corrente e la casa resta "al buio".
Ma nell'ipotesi di corrente che va via e torna, quindi tutto nuovamente alimentato, il mini-pc resterebbe staccato e non raggiungibile. (a quel punto chiaramente la soluzione sarebbero le singole app native dei dispositivi direi...)

**Reply by Scialla** [reply]
No, perché mai? Il MiniPC lo imposti nel BIOS che al ritorno dell'alimentazione riparta; HA riparte da solo all'avvio (se così lo imposti) e tutto torna perfettamente operativo
